docs/csharp/language-reference/operators/conditional-operator.md

```diff
@@ -10,70 +10,76 @@ helpviewer_keywords:
   - "conditional operator (?:) [C#]"
 ms.assetid: e83a17f1-7500-48ba-8bee-2fbc4c847af4
 ---
-
 # ?: Operator (C# Reference)
 
-The conditional operator (`?:`), commonly known as the ternary conditional operator, returns one of two values depending on the value of a Boolean expression. Following is the syntax for the conditional operator.  
+The conditional operator `?:`, commonly known as the ternary conditional operator, evaluates one of two expressions and returns its result depending on the value of a Boolean expression. Beginning with C# 7.2, the [conditional ref expression](#conditional-ref-expression) returns the reference to the result of one of the two expressions.
```

Comment:
This isn't quite right. It evaluates a Boolean expression, and returns the result of evaluating one of two expressions, depending on whether the Boolean expression is \`true\` or \`false\`.